packages/vuetify/src/components/VDivider/VDivider.tsx

```diff
@@ -2,20 +2,22 @@
 import './VDivider.sass'
 
 // Utilities
-import { computed } from 'vue'
+import { computed, toRef} from 'vue'
 import { convertToUnit, defineComponent } from '@/util'
 
 // Composables
 import { makeThemeProps, provideTheme } from '@/composables/theme'
+import { useBackgroundColor } from '@/composables/color'
 
 // Types
-type DividerKey = 'borderRightWidth' | 'borderTopWidth' | 'maxHeight' | 'maxWidth'
+type DividerKey = 'borderRightWidth' | 'borderTopWidth' | 'maxHeight' | 'maxWidth' | 'color' 
```

Comment:
```suggestion
type DividerKey = 'borderRightWidth' | 'borderTopWidth' | 'maxHeight' | 'maxWidth'
```
this change isn't necessary

Reply:
You are right, that was about a previous change which updates the "dividerStyles", but after working on the background color update, I forgot to remove this one.